wordpress-mobile/WordPress-iOS
# Core Data + Generics

Closes #6103 #4489

### To test:
- Verify that `CoreDataHelper` makes sense!
- Run `CoreDataHelperTests` and make sure everything is smooth

Needs review: @astralbodies 

May i bug you yet again Aaron?
Thanks!!
